### ISSUE TITLE
[Feature Request] Ability to drag an element off the screen to remove it from the navbar

### ISSUE THREAD

**Issue by puterboy** [jneilliii/OctoPrint-DragonOrder#23]
This would be a helpful feature to declutter the navbar.

Presumably you would just need to move the selected elements to the 'disabled:' entry in config.yaml

**Comment by jneilliii** [comment]
I'm starting to feel like you're trolling me @puterboy with all these feature requests. Your original issue seemed to indicate that disabling plugins didn't work, was that not the case, hence the reason for the edit? I think you might want to look into UI Customizer plugin as it has loads more options for hiding component and moving things around. It's like having several of my plugins combined with Themeify.

**Comment by puterboy** [comment]
Absolutely not!!!
I love your stuff -- and I only spend time commenting on things I
love. Your plugins are truly fantastic and my only interest in
suggesting enhancements is to make them even better for me as well as others.
You are of course welcome to ignore my suggestions.

The reason I edited the post is that 'disabling' only worked after
restarting Octoprint -- I thought I had and I also assumed that since
the GUI order changes took effect without a reboot that the same would
hold for deletions. It was late and hadn't realized you already saw
it, so rather than posting again, I just edited it.

The ability to delete an item should be easy as it just involves
moving it to the delete category... but it's not something I need
personally as I am comfortable editing config.yaml

Thanks again for all your hard work!!!

jneilliii wrote at about 09:32:51 -0700 on Tuesday, August 9, 2022:
